lib/signaling/v2/peerconnection.js

```diff
@@ -677,15 +680,10 @@ PeerConnectionV2.prototype.getRemoteMediaStreamTracksAndDataTrackReceivers = fun
  * @returns {Array<MediaStreamTrack>} mediaStreamTracks
  */
 PeerConnectionV2.prototype.getRemoteMediaStreamTracks = function getRemoteMediaStreamTracks() {
-  if (this._peerConnection.getReceivers) {
-    return this._peerConnection.getReceivers().map(function(receiver) {
-      return receiver.track;
-    }).filter(function(track) {
-      return track;
-    });
-  }
-  return flatMap(this._peerConnection.getRemoteStreams(), function(mediaStream) {
-    return mediaStream.getTracks();
+  return this._peerConnection.getReceivers().map(function(receiver) {
+    return receiver.track;
+  }).filter(function(track) {
+    return track;
   });
```

Comment:
Probably worth including a comment that explains that, the reason we filter here, is because there was a short-lived bug in Chrome where this could be null. I should've included such a comment the first time around.